packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx

```diff
@@ -1,13 +1,46 @@
 import * as React from 'react';
-import { css } from 'office-ui-fabric-react/lib/Utilities';
 import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
 import { List, ScrollToMode } from 'office-ui-fabric-react/lib/List';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
 import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
 import { createListItems, IExampleItem } from '@uifabric/example-data';
-import * as styles from './List.Scrolling.Example.scss';
+import { mergeStyleSets, getTheme, normalize } from 'office-ui-fabric-react/lib/Styling';
+
+const theme = getTheme();
+
+const styles = mergeStyleSets({
+  container: {
+    overflow: 'auto',
+    maxHeight: 500,
+    marginTop: 20,
+    selectors: {
+      '.ms-List-cell:nth-child(odd)': {
+        height: 50,
+        lineHeight: 50,
+        background: theme.palette.neutralLighter
+      },
+      '.ms-List-cell:nth-child(even)': {
+        height: 25,
+        lineHeight: 25
+      }
+    }
+  },
+  itemContent: [
+    theme.fonts.medium,
+    normalize,
+    {
+      position: 'relative',
+      display: 'block',
+      borderLeft: '3px solid ' + theme.palette.themePrimary,
+      paddingLeft: 27
+    }
+  ],
+  selected: {
```

Comment:
I don't see `selected` being used in this example.

Reply:
Fixed!